config/qualifications.yml

```diff
@@ -19,7 +19,7 @@ description: >
   Work with another developer to set up the following items:
 subtasks:
   - Use [`strap`](https://strap.umasstransit.it/) to configure macOS
-  - Install [`mariadb`](https://formulae.brew.sh/formula/mariadb)
+  - Install [`mysql`](https://formulae.brew.sh/formula/mysql)
```

Comment:
I wonder if this can be skipped at this point?

round-three has a `Brewfile` (that, admittedly specifies MariaDB). It might be easiest to stick with sqlite to start with (which is what [the official getting started guide does](https://guides.rubyonrails.org/getting_started.html).

Alternately, perhaps we could write-up a

Reply:
In terms of training, I loosely planned on having them use SQLite for their project apps anyways

I included this for dev env purposes as we're gonna end up on mysql everywhere eventually so independent of this that Brewfile should be updated...

...or since we're already maintaining a cask for rbenv and nodenv, does it make sense for us to roll this in there (along with other, arguably more general libs that we need to build ruby/rails like openssl, psych and whatever the image processing lib is - gmp I think?)

Reply:
https://github.com/umts/homebrew-umts-dev/pull/11

If so